### ISSUE TITLE
Dual spec in german wrong codec

### ISSUE THREAD

**Issue by V-Cyberpunk** [azerothcore/azerothcore-wotlk#18773]
### Current Behaviour

in german, there is the word "möchtet" not correct.
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/44802344/444dc9fd-f9e7-4042-a6c8-5a35b7158fea)

### Expected Blizzlike Behaviour

correct display of "möchtet"

### Steps to reproduce the problem

go to trainer and ask for learning dual spec

### AC rev. hash/commit

5386443

### Operating system

Debian 12 x64

### Custom changes or Modules

BreakingNewsOverride
ExtendedXP
mod-1v1-arena
mod-account-achievements
mod-account-mounts
mod-ah-bot
mod-anticheat
mod-autobalance
mod-boss-announcer
mod-duel-reset
mod-dynamic-xp
mod-eluna
mod-emblem-transfer
mod-gain-honor-guard
mod-instance-reset
mod-learn-spells
mod-npc-beastmaster
mod-npc-buffer
mod-npc-enchanter
mod-pvp-titles
mod-rdf-expansion
mod-reagent-bank
mod-solocraft
mod-solo-lfg
mod-starter-guild
mod-TimeIsTime
mod-transmog
mod-weekend-xp
mod-world-chat
ReusablePotion

**Comment by chaosua** [comment]
is it possible FONTS problem in client ?

**Comment by V-Cyberpunk** [comment]
no. issue is in the gossip_menu_option_locale table i figured out. will post pr later.
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/44802344/0e6509a1-0438-482e-ab0e-6b5c1909a153)
